src/renderer/containers/ReplayFileStats/GameProfileHeader.tsx

```diff
@@ -182,27 +189,68 @@ const GameDetails: React.FC<{
   if (duration === null || duration === undefined) {
     duration = _.get(stats, "lastFrame");
   }
+
   const durationLength =
-    duration !== null && duration !== undefined ? convertFrameCountToDurationString(duration, "m[m] ss[s]") : "Unknown";
+    duration !== null && duration !== undefined
+      ? file.settings.gameMode == GameMode.TARGET_TEST && file.metadata
+        ? frameToGameTimer(file.metadata?.lastFrame as number, file.settings)
+        : convertFrameCountToDurationString(duration, "m[m] ss[s]")
+      : "Unknown";
+
+  const distance = _.get(stadiumStats, "distance");
+  const units = _.get(stadiumStats, "units");
+
+  const eventDisplay = {
+    label: <EventIcon />,
+    content: monthDayHourFormat(moment(startAtDisplay)) as string,
+  };
+
+  const timerDisplay = {
+    label: <TimerIcon />,
+    content: durationLength,
+  };
+
+  const stageDisplay = {
+    label: <LandscapeIcon />,
+    content: stageName,
+  };
 
-  const displayData = [
-    {
-      label: <EventIcon />,
-      content: monthDayHourFormat(moment(startAtDisplay)) as string,
-    },
-    {
-      label: <TimerOutlinedIcon />,
-      content: durationLength,
-    },
-    {
-      label: <LandscapeIcon />,
-      content: stageName,
-    },
-    {
-      label: <SportsEsportsIcon />,
-      content: platform,
-    },
-  ];
+  const platformDisplay = {
+    label: <SportsEsportsIcon />,
+    content: platform,
+  };
+
+  const targetTestDisplay = {
+    label: <TrackChangesIcon />,
+    content: "Break the Targets",
+  };
+
+  const homerunDisplay = {
+    label: <SportsCricket />,
+    content: "Home Run Contest",
+  };
+
+  const distanceDisplay = {
+    label: <Straighten />,
+    content: `${distance} ${units}`,
+  };
+
+  const gameMode = file.settings.gameMode;
+
+  let displayData;
```

Comment:
```suggestion
  let displayData: { label: React.ReactNode, content: React.ReactNode }[];
```
Please don't use `any`. (`let blah;` types it as `any`).